### DISCUSSION TITLE
ScrollToIndex scroll behavior smooth dont work in safari ios

### DISCUSSION THREAD

**Discussion by kimixz** [petyosi/react-virtuoso#402] [Q&A]
**Describe the bug**
In general scroll behavior smooth doesn't work on safari ios. However, [smoothscroll-polyfill](https://github.com/iamdustan/smoothscroll) is a library that add support for using smooth scroll behavior in safari and other browsers. But this library doesn't work on scrollToIndex method.How can i fix this problem?

**Comment by petyosi** [comment]
This is a question suitable for the polyfill library maintainer. For what is worth, here's the point where `scrollToIndex` gets translated into a DOM `scrollTo` call: https://github.com/petyosi/react-virtuoso/blob/f91b307cfa332f9ddaa42e33b039834a44dfcb1a/src/hooks/useScrollTop.ts#L102.
